x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/260_rule_query_search.yml

```diff
@@ -236,4 +236,154 @@ setup:
   - match: { hits.total.value: 1 }
   - match: { hits.hits.0._id: 'doc1' }
 
+---
+"Perform a rule query with an organic query that must be rewritten to another query type":
+  - skip:
+      version: " - 8.12.2"
```

Comment:
This might fail tests until it's backported, if tests fail I'll adjust to 8.12.99 and fix in a subsequent backport.